src/test/java/com/epam/tc/hw3/ex1/SiteTestJdiLightFirst.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.tc.hw3.ex1;
+
+import com.epam.tc.hw3.MainTest;
+import org.testng.annotations.Test;
+
+public class SiteTestJdiLightFirst extends MainTest {
```

Comment:
We should name the test classes and test methods according to what we're checking in them, please, try to think of better namings